### PR TITLE
Java Script regular expressions, errors and  " this "

### PR DESCRIPTION
**Pull request by ioandodita069**

### CODE REVIEW

File: js/regular-expressions.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+var obj = /r/.exec("Mos Craciun nu exista");
```
Comment (pancudaniel7) [inline js/regular-expressions.js:1]: true!

File: js/problem.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
+test("Ana are ce nu are Maria.");
+function test(e) {
+    var words = e
+        .toLowerCase()
+        .split("")
+
+    var consonant = words
```
Comment (pancudaniel7) [inline js/problem.js:7]: Consonant?

File: js/problem.js [hunk 3]
```diff
@@ -0,0 +1,16 @@
+test("Ana are ce nu are Maria.");
+function test(e) {
```
Comment (pancudaniel7) [inline js/problem.js:2]: Name should describe the inner code

**Review by pancudaniel7 — CHANGES_REQUESTED** [review]

File: js/problem.js (hunk 2 above)
Comment (ioandodita069) [reply js/problem.js:7]: nici eu nu stiam,urat tare

**Pull request merged**